scripts/migrations/11-gas-price-fix.py

```diff
@@ -0,0 +1,20 @@
+"""
+Changes gas price computation costs. No state migration needed for this change
+
+"""
+
+import sys
+import os
+import json
+from collections import OrderedDict
+
+home = sys.argv[1]
+output_home = sys.argv[2]
+
+config = json.load(open(os.path.join(home, 'output.json')), object_pairs_hook=OrderedDict)
+
+assert config['protocol_version'] == 10
+
+config['protocol_version'] = 11
+
```

Comment:
You need to overwrite the old values. See how it is done for example in `migrate/scripts/6-state-stake.py`